src/cc/dummy.rs

```diff
@@ -0,0 +1,177 @@
+// Copyright (C) 2019, Cloudflare, Inc.
```

Comment:
Already mentioned in the PR description. "This is exactly same as Reno for now and works as a template of congestion control module and also demonstrate that there can be multiple CC modules.". I need this to test if I can switch over between different CCs. In the future it can be removed/replaced.

Reply:
Well. I removed `dummy`.